web3/eth.py

```diff
@@ -187,6 +188,10 @@ def getTransactionFromBlock(self, block_identifier, transaction_index):
             [block_identifier, transaction_index],
         )
 
+    def waitForTransactionReceipt(self, transaction_hash):
```

Comment:
I would expect this to have a timeout, like the underlying method.

Reply:
you got it 👍